lld/test/ELF/invalid/broken-relaxation-x64.test

```diff
@@ -44,4 +44,132 @@ Symbols:
     Value:    0x12345
     Size:     4
     Binding:  STB_GLOBAL
- 
+
+
+# RUN: yaml2obj --docnum=2 %s -o %t2.o
```

Comment:
Thanks for adding negative tests but YAML is not very readable. You can use assembly tests with `.reloc ., R_...`. See x86-64-reloc-32.s

Reply:
Is the new assembly test okay?